src/main/java/nl/knaw/dans/filemigration/api/Expected.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (C) 2021 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.filemigration.api;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "expected")
+public class Expected {
+
+  @Id
+  @Column(name="doi")
+  private String doi;
+  @Column(name="sha1checksum")
+  private String sha1checksum;
+  @Column(name="easy_file_id")
+  private String easyFileId;
+  @Column(name="fsRdbPath")
```

Comment:
Why not keep up the snake_case convention for column names; you are switching to camelCase for some reason...?